packages/sheets/src/commands/commands/set-style.command.ts

```diff
@@ -75,8 +75,8 @@ export const SetStyleCommand: ICommand<ISetStyleCommandParams<unknown>> = {
         const target = getSheetCommandTarget(univerInstanceService);
```

Comment:
You should pass params to `getSheetCommandTarget`.

Reply:
And `ISetStyleCommonParams` should extends `ISheetCommandSharedParams`.

Reply:
changed